consensus/istanbul/backend/api.go

```diff
@@ -474,6 +477,100 @@ func (api *APIExtension) GetBlockWithConsensusInfoByHash(blockHash common.Hash)
 	return api.makeRPCBlockOutput(block, cInfo, block.Transactions(), receipts), nil
 }
 
+func (api *APIExtension) GetBlsInfos(number rpc.BlockNumber) (map[string]interface{}, error) {
+	kip113Addr, err := api.GetActiveAddressFromRegistry(system.Kip113Name, number)
+	if err != nil {
+		return nil, err
+	}
+
+	var bn *big.Int
+	if number == rpc.LatestBlockNumber || number == rpc.PendingBlockNumber {
+		bn = big.NewInt(api.chain.CurrentBlock().Number().Int64())
+	} else {
+		bn = big.NewInt(number.Int64())
+	}
+
+	backend := backends.NewBlockchainContractBackend(api.chain, nil, nil)
+	if infos, err := system.ReadKip113All(backend, kip113Addr, bn); err != nil {
+		return nil, err
+	} else {
+		blsInfos := make(map[string]interface{})
```

Comment:
minor comment, but could you follow the error handling convention in other functions?
```suggestion
	infos, err := system.ReadKip113All(backend, kip113Addr, bn)
	if err != nil {
		return nil, err
	}
	
	blsInfos := make(map[string]interface{})
```

Reply:
Thanks. Update at https://github.com/klaytn/klaytn/pull/2111/commits/d26e6352f2868c2f6362708ec97088bb3b74fe7e